cl/opinion_page/templates/recap_document.html

```diff
@@ -100,9 +101,16 @@ <h4>Description</h4>
                  aria-controls="profile"
                  role="tab"
                  data-toggle="tab">Text</a></li>
+          {% if authorities_context.total_authorities_count %}
+            <li class="{% block nav-authorities %}inactive{% endblock %}" role="presentation">
+              <a href="{{ authorities_context.view_all_url }}">
+                Authorities
+              </a>
```

Comment:
Since we already know the count, why not do:

```suggestion
              <a href="{{ authorities_context.view_all_url }}">
                {{ authorities_context.total_authorities_count|intcomma }}Authorit{{ authorities_context.total_authorities_count|pluralize:"y,ies" }}
              </a>
```

Reply:
Actually, sorry, I take this back. Wouldn't it be better and more performant to just do an `exists()` call to see if there are authorities, instead of getting and counting them (like we do on the entries and parties tabs)?

Reply:
> Wouldn't it be better and more performant to just do an exists() call to see if there are authorities, instead of getting and counting them (like we do on the entries and parties tabs)?

You're correct! I'm updating this template.